src/valid/interface.rs

```diff
@@ -571,8 +591,10 @@ impl super::Validator {
                     self.bind_group_masks.push(BitSet::new());
                 }
                 if !self.bind_group_masks[bind.group as usize].insert(bind.binding as usize) {
-                    return Err(EntryPointError::BindingCollision(var_handle)
-                        .with_span_handle(var_handle, &module.global_variables));
+                    if self.flags.contains(super::ValidationFlags::BINDINGS) {
```

Comment:
IIRC, both of these are already inside the feature = "validate" from the high above
also, if we haven't already, we need CI to check the configuration without the feature, and it would complain

Reply:
My bad, they are indeed covered by the enclosing items.

> also, if we haven't already, we need CI to check the configuration without the feature, and it would complain

Yeah, as far as I see we don't check code compiles in CI without the feature (filed https://github.com/gfx-rs/naga/issues/2173).